src/screens/Departures/StopPlaceView.tsx

```diff
@@ -115,6 +117,7 @@ export default function StopPlaceView({
               />
             </View>
           )}
+          {extraHeaderView && extraHeaderView()}
```

Comment:
Cant this instead be a new mode, like `'Map'` to enable the To/From buttons? Should first try to reuse existing configuration possibilities instead of adding new ones.

Reply:
Fixed! ;), however, what I had in mind was to not add more complexity to the stop place and injected extra components from outside.